t2kdm/configuration.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, filename=None, defaults=default_values):
         """
 
         # Load the default values
-        for key, val in defaults.items():
+        for key, val in list(defaults.items()):
```

Comment:
This `list` is not needed, since we directly iterate over the return value of `items`.